Sources/PointFree/Emails/SharedEmailComponents.swift

```diff
@@ -31,13 +32,41 @@ let emailFooterView = View<Prelude.Unit> { _ in
 
           p([`class`([Class.pf.type.body.small])], [
             "Our postal address: 139 Skillman #5C, Brooklyn, NY 11211"
-            ])
-          ])
+            ]),
+
+          ] + unsubscribeView.view((user, newsletter)))
         ])
       ])
     ])
 }
 
+private let unsubscribeView = View<(Database.User?, Database.EmailSetting.Newsletter?)> { user, newsletter -> [Node] in
+  guard let user = user, let newsletter = newsletter else { return [] }
+
+  return [
+    p([`class`([Class.pf.type.body.small])], [
+      .text(encode(subscribedReason(newsletter: newsletter))),
+      "If you no longer wish to receive emails like this, you can unsubscribe ",
+      a([href(url(to: .expressUnsubscribe(userId: user.id, newsletter: newsletter)))], ["here"]),
```

Comment:
again cool to see how easy it is to use the router, when under the hood its encrypting this data so that the user cant tamper with it